### PR TITLE
Update set-up-monitoring.md

### PR DESCRIPTION
**Pull request by JStickler**
Fix typo in admonition

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Comment by joe-elliott** [comment]
howdy @JStickler. Seems like you need to sign the CLA. Then we can kick CI til it passes and merge.

**Review by knylander-grafana — APPROVED** [review]
Thank you for fixing the typos!

File: docs/sources/tempo/operations/monitor/set-up-monitoring.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The steps outlined below use the Grafana Agent Flow configurations described in
 {{< admonition type="note" >}}
 Update any instructions in this document for your own deployment.
 
-If you use the [Kubernetes integration Grafana Agent Helm chart](https://grafana.com/docs/agent/latest/flow/get-started/install/kubernetes/), you’ll be able to use the Kubernetes scrap annotations to automatically scrap Tempo. You’ll need to add the labels to all of the deployed components.
+If you use the [Kubernetes integration Grafana Agent Helm chart](https://grafana.com/docs/agent/latest/flow/get-started/install/kubernetes/), you’ll be able to use the Kubernetes scrape annotations to automatically scrape Tempo. You’ll need to add the labels to all of the deployed components.
 {{% /admonition %}}
```
Comment (knylander-grafana) [inline docs/sources/tempo/operations/monitor/set-up-monitoring.md:29]:
```suggestion


```

**Comment by knylander-grafana** [comment]
I can't figure out the issue this PR had with the drone deb package build failing. I'm going to go ahead and merge the PR since it's only a two-character change.

**Pull request merged**